### DISCUSSION TITLE
What if local and remove folders of GitHub Desktop aren't the same?

### DISCUSSION THREAD

**Discussion by adslbarxatov** [community/community#23730] [New to GitHub]
<p>Hi!</p>
<p>What if local and remove folders of GitHub Desktop aren’t the same? We mean, how can we do that? F.e., we need to commit local directories <em>Sources</em> and <em>Generics</em> into remote directory <em>src</em>. Is it possible?</p>
<p>Thanks!</p>

**Comment by francisfuzz** [comment]
<aside class="quote no-group" data-username="adslbarxatov" data-post="1" data-topic="137132">
<div class="title">
<div class="quote-controls"></div>
 adslbarxatov:</div>
<blockquote>
<p>What if local and remove folders of GitHub Desktop aren’t the same? We mean, how can we do that? F.e., we need to commit local directories <em>Sources</em> and <em>Generics</em> into remote directory <em>src</em> . Is it possible?</p>
</blockquote>
</aside>
<p>:wave:@adslbarxatov: When you say “remove”, do you mean “remote” or something else?</p>

**Comment by adslbarxatov** [comment, marked as answer]
<p>Thanks for your answer!</p>
<p>Yes, <em>remote</em>. It’s mistype.</p>
<p>We found a workaround: we’re using .cmd that copies required files from different local directories into local <em>src</em> directory. And this <em>src</em>, in its turn, is a part of local clone of repository that we’re using for performing commits. But we almost sure that there must be built-in way to do this</p>
